react/.npmignore

```diff
@@ -1,3 +1,3 @@
-node_modules/
```

Comment:
i think this needs to be there, so that we don't package node_modules with the publish

Reply:
I noticed we don’t have it in master and we don’t have it for vanilla or angular. Should we add it?